RallyHereDebugTool/Source/Private/RHDTW_PlayerSettings.cpp

```diff
@@ -144,6 +152,61 @@ void FRHDTW_PlayerSettings::DoModifySettings()
 	}
 }
 
+void FRHDTW_PlayerSettings::DoSettingsTypes()
+{
+	auto* GI = GetGameInstance();
+	if (GI == nullptr)
+	{
+		ImGui::Text("%s", "Could not find owning game instance.");
+		return;
+	}
+
+	auto* pGISS = GI->GetSubsystem<URH_GameInstanceSubsystem>();
+	if (pGISS == nullptr)
+	{
+		ImGui::Text("%s", "URH_GameInstanceSubsystem not available.");
+		return;
+	}
+
+	URH_SettingsSubsystem* pRH_SettingsSubsystem = pGISS->GetSettingsSubsystem();
+	if (pRH_SettingsSubsystem == nullptr)
+	{
+		ImGui::Text("%s", "URH_SettingsSubsystem not available.");
+		return;
+	}
+
+	if (ImGui::Button("Get Setting Types"))
+	{
+		pRH_SettingsSubsystem->GetSettingTypes();
+	}
+	ImGui::Separator();
+
+	if (pRH_SettingsSubsystem->GetCachedSettingTypes().Num() > 0)
+	{
+		for (const auto& Pair : pRH_SettingsSubsystem->GetCachedSettingTypes())
+		{
+			if (ImGui::TreeNodeEx(TCHAR_TO_UTF8(*Pair.Key), RH_DefaultTreeFlagsDefaultOpen))
+			{
+				for (const auto& VersionPair : Pair.Value.GetVersions())
+				{
+					if (ImGui::TreeNodeEx(TCHAR_TO_UTF8(*FString::Printf(TEXT("%s##%s"), *VersionPair.Key, *Pair.Key)), RH_DefaultTreeFlagsLeaf | ImGuiTreeNodeFlags_DefaultOpen))
```

Comment:
Anh added a ImGuiDisplayJsonObject() utility function for displaying JSON blobs consistently.